rocket-mission-angular/src/app/components/missions-list/missions-list.component.html

```diff
@@ -44,7 +44,7 @@
       <th mat-header-cell *matHeaderCellDef></th>
       <td mat-cell *matCellDef="let mission">
         <button mat-flat-button color="primary" (click)="accept()" >{{'MISSION_LIST.ACCEPT'|translate}}</button>
-        <button mat-flat-button class="ms-2" (click)="reject()" color="warn">{{'MISSION_LIST.REJECT'|translate}}</button>
+        <button mat-flat-button class="ms-2" (click)="reject()" (mouseleave)="disableRouting=false" (mouseenter)="disableRouting=true" color="warn">{{'MISSION_LIST.REJECT'|translate}}</button>
```

Comment:
you need to store the previous value of disableRouting on mouse enter and use that stored value on mouse leave. Now, this enables routing on a table where routing was disabled.